cves/kernel/CVE-2016-9685.yml

```diff
@@ -278,8 +295,11 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    IPC was not directly used in the affected code. However, the issue existed 
+    in the logging mechanism of the xfs filesystem so it can be assumed to 
+    have had some effect over IPC.
```

Comment:
The phrasing of the end here is a little awkward. Probably not a huge deal but might be worth rephrasing to be more specific as to the effect